### ISSUE TITLE
Better functional.dropout{,2d,3d} docs

### ISSUE THREAD

**Issue by zou3519** [pytorch/pytorch#4177]
http://pytorch.org/docs/master/nn.html#torch.nn.functional.dropout
